demos/common/python/openvino/model_zoo/model_api/models/image_model.py

```diff
@@ -121,8 +146,16 @@ def preprocess(self, inputs):
         return dict_inputs, meta
 
     def _change_layout(self, image):
+        '''Changes the input image layout to fit the layout of the model input layer.
+
+        Args:
+            inputs (ndarray): a single image as 3D array in HWC layout
+
+        Returns:
+            - the image with layout aligned with the model layout
+        '''
         if self.nchw_layout:
-            image = image.transpose((2, 0, 1))  # Change data layout from HWC to CHW
+            image = image.transpose((2, 0, 1))  # Change layout from HWC to CHW
```

Comment:
You can be even more comprehensive:
```suggestion
            image = image.transpose((2, 0, 1))  # HWC->CHW
```

Reply:
Done